pinot-timeseries/pinot-timeseries-spi/src/main/java/org/apache/pinot/tsdb/spi/series/builders/MaxTimeSeriesBuilder.java

```diff
@@ -39,7 +39,7 @@ public MaxTimeSeriesBuilder(String id, TimeBuckets timeBuckets, List<String> tag
 
   @Override
   public void addValueAtIndex(int timeBucketIndex, Double value) {
```

Comment:
Suggest adding `@Nullable` annotation both to base class and the overrides

Reply:
Ack. Will add it in the next PR I am working on.